client/components/RelatedItems/Outfit.jsx

```diff
@@ -1,19 +1,60 @@
 import React from 'react';
 import CardTemplate from './CardTemplate.jsx';
 
-const Outfit = (props) => {
-  return (
-    <div className="container">
-      <h3 className="related-products">Outfit</h3>
-      <div className="carousel">
-        <ul className="slider">
-          {props.selectedItemsList.map(product=>
-            <CardTemplate key={product.id} product={product} />
-            )}
-        </ul>
-      </div>
-    </div>
-  )
+class Outfit extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      outfitAdded: false
+    }
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({
+      outfitAdded: true
+    })
+    this.props.addToOutfit();
+  }
+
+  render() {
+    if (!this.state.outfitAdded) {
+      return (
+        <div className="container">
+          <h3 className="related-products">YOUR OUTFIT</h3>
+          <ul className="carousel">
+            <li className="card empty-card">
+              <button className="add-button" onClick={this.handleClick}>+</button>
+              Add to Outfit
+            </li>
+          </ul>
+          <div className="button-wrapper">
+            <button className="slideLeft">Left</button>
+            <button className="slideRight">Right</button>
+          </div>
+        </div>
+      )
+    } else {
+      return (
+        <div className="container">
+          <h3 className="related-products">YOUR OUTFIT</h3>
+          <ul className="carousel">
+            <li className="card empty-card">
+                <button className="add-button" onClick={this.handleClick}>+</button>
+                Add to Outfit
+            </li>
+            {this.props.selectedItemsList.map(product=>
+              <CardTemplate key={product.id} product={product} />
+              )}
+          </ul>
+          <div className="button-wrapper">
+            <button className="slideLeft">Left</button>
+            <button className="slideRight">Right</button>
+          </div>
+        </div>
+      )
+    }
```

Comment:
seems to me the 2 conditions render the same thing except for where you map `props.selectedItemsList`. If your parent component is sure to pass an array, which could be empty, then the conditional rendering shouldn't be necessary.

Reply:
I was going to refactor the rendering of the Outfit component next to get rid of some duplicate elements, but you are so right, I just realized that I don't even need the conditional rendering, to begin with. Thank you for the great suggestion!